examples/helloElektra.c

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file
+ *
+ * @brief
+ *
+ * @copyright BSD License (see LICENSE.md or https://www.libelektra.org)
+ */
+
+#include <kdb.h>
+
+#include <stdio.h>
+
+int main (void)
+{
+	KeySet * config = ksNew (0, KS_END);
+	Key * root = keyNew ("system/test", KEY_END);
+
+	printf ("Open key database\n");
+	KDB * handle = kdbOpen (root);
+
+	printf ("Retrieve key set\n");
+	kdbGet (handle, config, root);
+
+	printf ("Number of mappings: %zu\n", ksGetSize (config));
+
+	Key * key = keyNew ("system/test/hello", KEY_VALUE, "elektra", KEY_END);
+	printf ("Add new key %s\n", keyName (key));
+	ksAppendKey (config, key);
+	printf ("Number of mappings: %zu\n", ksGetSize (config));
+	printf ("\n%s, %s\n\n", keyBaseName (key), keyString (key));
+
+	printf ("Delete mappings\n");
```

Comment:
mention here that it will only be deleted in-memory?

Reply:
I changed the description to [“Delete mappings inside memory”](https://github.com/ElektraInitiative/libelektra/pull/1600/commits/f8b62595f6fadf266f088bfa4065cd02d0b208c1#diff-8cdb83a7d91bc9a7edc69f6a8f01d4ceR36) and also added a description on how to [store the data on disk](https://github.com/ElektraInitiative/libelektra/pull/1600/commits/f8b62595f6fadf266f088bfa4065cd02d0b208c1#diff-8cdb83a7d91bc9a7edc69f6a8f01d4ceR32).